internal/forms/forms.go

```diff
@@ -1011,6 +1019,8 @@ func (b formMessageBuilder) initFormValues() {
 }
 
 func (b formMessageBuilder) scanTmplBuildMessage(tmplPath string) (MessageForm, error) {
+	inBody := false
```

Comment:
Minor detail, but I would rather this be defined just before the `for` loop since this is where it is used.